conf/flight_plans/demo_gvf_rotorcraft.xml

```diff
@@ -0,0 +1,104 @@
+    <variable var="nominal_alt" init="0.5" type="float" min="0." max="10." step="0.1"/>
+  </variables>
+
+  <blocks>
+    <block name="FPInit">
+      <call_once fun="NavKillThrottle()"/>
+      <while cond="!GpsFixValid()"/>
+      <while cond="TRUE"/>
+    </block>
+
+    <block name="Holding point">
+      <call_once fun="NavKillThrottle()"/>
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+    </block>
+
+    <block name="Start Engine" strip_button="Start Engine" strip_icon="resurrect.png" group="home">
+      <call_once fun="NavResurrect()"/>
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+    </block>
+
+    <block name="Take off" strip_button="Takeoff" strip_icon="takeoff.png" group="home">
+      <call_once fun="NavVerticalAltitudeMode(nominal_alt, 0.0)"/>
+    </block>
+
+    <block name="Standby" strip_button="Standby" strip_icon="home.png" group="home">
+      <stay wp="STDBY"/>
+    </block>
+
+    <!-- Vars found in sw/airborne/firmwares/rotorcraft/navigation.h --> 
+    <block name="Move East">
+      <while cond="1">
+        <set var="nav.setpoint_mode" value="NAV_SETPOINT_MODE_SPEED"/>
+        <set var="nav.speed.x" value="3.0"/>
+        <set var="nav.speed.y" value="0.0"/>
+      </while>
+    </block>
+
+    <block name="GVF Segment 1">
+      <call fun="gvf_segment_loop_wp1_wp2(WP_S1_START, WP_S1_END, gvf_segment_par.d1, gvf_segment_par.d2)"/>
+    </block>
+
+    <block name="GVF Segment 2">
+      <call fun="gvf_segment_loop_wp1_wp2(WP_S2_START, WP_S2_END, gvf_segment_par.d1, gvf_segment_par.d2)"/>
+    </block>
+
+    <!-- Can't go below 1 because int var -->
+    <block name="GVF 1m Circle">
+      <call fun="gvf_ellipse_wp(WP_TD, 1, 1, gvf_ellipse_par.alpha)"/>
+    </block>
+
+    <block name="GVF 25m Circle">
+      <call fun="gvf_ellipse_wp(WP_TD, 25, 25, gvf_ellipse_par.alpha)"/>
+    </block>
+
+    <!-- Function found in ./sw/airborne/modules/guidance/gfv/gvf.h -->
+    <block name="GVF Ellipse">
+      <call fun="gvf_ellipse_wp(WP_TD, gvf_ellipse_par.a, gvf_ellipse_par.b, gvf_ellipse_par.alpha)"/>
+    </block>
+
+    <block name="Land here" key="l" strip_button="Land Here" strip_icon="land-right.png">
+      <call_once fun="NavSetWaypointHere(WP_TD)"/>
+    </block>
+
+    <block name="Land">
+      <go wp="TD"/>
+    </block>
+
+    <block name="Land Target" strip_button="Land Target" group="target">
+      <exception cond="GetPosHeight() @LT 0.3" deroute="Ramp down"/>
+      <exception cond="!nav_is_in_flight()" deroute="Kill Engine"/>
+      <stay climb="nav.descend_vspeed" vmode="climb" wp="TD"/>
+    </block>
+
+    <block name="Ramp down" key="r">
+      <exception cond="guidance_v.rc_delta_t @LT 0.1*9600." deroute="Kill Engine"/>
+      <set var="fp_throttle" value="guidance_v.rc_delta_t/9600."/>
+      <stay throttle="fp_throttle-0.1*stage_time" vmode="throttle" wp="TD"/>
+    </block>
```

Comment:
You should remove this block (and the related exception in previous block). I doubt that it is really working since the last changes in guidance. I know that it comes from one of our example, but we really have too fix that as well.

Reply:
Ok!